sources/kernel/.build.mk

```diff
@@ -5,11 +5,13 @@ KERNEL_C_SRC+=$(wildcard sources/kernel/*.c) \
 KERNEL_S_SRC+=$(wildcard sources/kernel/$(CONFIG_ARCH)/*.s)
 
 KERNEL_LIBS_SRC = \
+			  $(wildcard sources/libraries/ansi/*.c) \
 			  $(wildcard sources/libraries/ubsan/*.c) \
 			  $(wildcard sources/libraries/brutal/io/fmt.c) \
 			  $(wildcard sources/libraries/brutal/io/print.c) \
 			  $(wildcard sources/libraries/brutal/io/scan.c) \
 			  $(wildcard sources/libraries/brutal/io/write.c) \
+			  $(wildcard sources/libraries/brutal/io/buffer.c) \
 			  $(wildcard sources/libraries/brutal/mem/*.c) \
 			  $(wildcard sources/libraries/brutal/text/*.c) \
 			  $(wildcard sources/libraries/brutal/sync/*.c) \
```

Comment:
Why are we including that into the kernel? Why the whole ANSI C library?

Reply:
for ctypes but I should not use the whole ANSI C lib :x